deploy/clowdapp.yaml

```diff
@@ -193,8 +196,8 @@ objects:
     - host-inventory
     - rbac
     - vmaas
-    # - ingress # These deps are failing, commenting temporarily
-    # - puptoo
+    - ingress # These deps are failing, commenting temporarily
+    - puptoo
```

Comment:
`puptoo` is probably not needed, it should be set as a dependency of `ingress`

Reply:
I think it's good to specify explicitly... bonfire resolves dependencies in a good way.